diagnostic.md

```diff
@@ -57,13 +64,17 @@ Please write a method that traverses from a list item (`li`) DOM node to the
 body DOM node, then hides it.
 
 <!-- your answer starts here -->
-
+$("li").on("click",()=>{
```

Comment:
This attaches a click handler to `<li>` elements.  We want to traverse from a `<li>` element to the `<body>` element and hide it.  One possible way is `$('li').parent().parent().hide();`.